### PR TITLE
bpo-39068 guard _b85chars2 initialization

### PR DESCRIPTION
**Pull request by drmonkeysee**
Under multi-threading scenarios a race condition may occur where a thread sees an initialized `_b85chars` table but an uninitialized `_b85chars2` table due to the guard only checking the first table.

This causes an exception like:

```
  File "/usr/lib/python3.6/base64.py", line 434, in b85encode
    return _85encode(b, _b85chars, _b85chars2, pad),
  File "/usr/lib/python3.6/base64.py", line 294, in _85encode
    for word in words],
  File "/usr/lib/python3.6/base64.py", line 294, in <listcomp>
    for word in words],
 "TypeError: 'NoneType' object is not subscriptable
```

https://bugs.python.org/issue39068

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@drmonkeysee

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Review by Commod0re — APPROVED** [review]

**Review by camhawkins — APPROVED** [review]

**Review by IEarle — APPROVED** [review]

File: Lib/base64.py [hunk 1]
```diff
@@ -430,6 +430,7 @@ def b85encode(b, pad=False):
     # if the function is never called
     if _b85chars is None:
```
Comment (serhiy-storchaka) [inline Lib/base64.py]: It is enough to check `if _b85chars2 is None` here.

These checks are cheap, but they are performed at every `b85encode()` call. Lesser checks is better.

`a85encode()` should be updated too. And may be other functions if they initialize two or more globals.

Comment (drmonkeysee) [reply Lib/base64.py]: done. visually scanned the file for other cases but only found the additional one in `a85encode()`.

**Comment by serhiy-storchaka** [comment]
Please add [a NEWS entry](https://devguide.python.org/committing/#what-s-new-and-news-entries). Since it is your first contribution, add also your name in Misc/ACKS.

**Review by serhiy-storchaka — APPROVED** [review]

**Comment by serhiy-storchaka** [comment]
Thank you for your contribution @drmonkeysee. Sorry for the delay, this ЗК fell out of my window of attention.

**Pull request merged**